airflow/providers/airbyte/hooks/airbyte.py

```diff
@@ -50,80 +46,63 @@ class AirbyteHook(HttpHook):
     conn_type = "airbyte"
     hook_name = "Airbyte"
 
-    RUNNING = "running"
-    SUCCEEDED = "succeeded"
-    CANCELLED = "cancelled"
-    PENDING = "pending"
-    FAILED = "failed"
-    ERROR = "error"
-    INCOMPLETE = "incomplete"
```

Comment:
Removed this to start using the enum class provided by the Airbyte API.

Reply:
Please add entry of 4.0.0 in the changelog of the provider with 1-2 paragraph explaining how users should migrate from previous version to the current one.